src/config.rs

```diff
@@ -52,6 +54,7 @@ impl Default for Config {
         Self {
             blocking_mode: false,
             idle_connection_timeout: Duration::from_secs(5),
+            heartbeat_interval: Duration::from_secs(2),
```

Comment:
I think we should probably, in the future, have this calculated based on the current network circumstances. With two seconds this is safe, we might as well move it up to 3, because the idling time is 5 seconds, then the heartbeat has a total time of 2 seconds to arrive at the client.

Reply:
Yes, I thought about that as well, but didn't want to add RTT calculations into it (for now). Although I agree that that is probably a good idea here.

I chose 2 seconds because then if one heartbeat packet is lost, we have enough time (probably) to send another one before the connection times out. Then again, this really depends on the `idle_connection_timeout` value of the receiver, so the user currently has to adjust this value to their wishes anyway.

Reply:
Right, I can understand that I am not too certain about our RTT calculations as well, so I am fine with having this statically typed now. And figure out a way to do it based on the current internet speed in the future.